src/sfProject.ts

```diff
@@ -284,13 +284,13 @@ export class SfProjectJson extends ConfigFile {
    * for packaging operations that want to do something for each package entry.
    */
   public getUniquePackageDirectories(): NamedPackageDir[] {
-    const visited: Dictionary<boolean> = {};
+    const visited: Set<string> = new Set<string>();
```

Comment:
maybe a bit more than the change intended, but couldn't this be rewritten as `return new Set<string>(this.getPacageDirectoriesSync())` 

we shouldn't need to worry about adding duplicate values to a Set

Reply:
The goal is to retain order if dir entries found in pjson. Placing contents into any collection would not guarantee original order.